tools/rpkg/R/dbConnect__duckdb_driver.R

```diff
@@ -9,6 +9,9 @@
 #' @param ... Ignored
 #' @param debug Print additional debug information such as queries
 #' @param read_only Set to `TRUE` for read-only operation
+#' @param tibble Whether a [tibble::tbl_df] should be returned rather than
+#'   a [data.frame]. This is recommended if you are working with lists and structs
+#'   as tibbles, unlike regular data frames, can easily handle nested data.
```

Comment:
Minor: nested columns are lists of data frames (or other vectors), structs would correspond to packed columns.

```suggestion
#'   as tibbles, unlike regular data frames, can easily handle nested and packed data.
```